src/plugins/data/public/search/search_service.ts

```diff
@@ -92,9 +92,9 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
     { application, http, injectedMetadata, notifications, uiSettings }: CoreStart,
     { fieldFormats, indexPatterns }: SearchServiceStartDependencies
   ): ISearchStart {
-    const search: ISearchGeneric = (request, options) => {
+    const search = ((request, options) => {
       return this.searchInterceptor.search(request, options);
-    };
+    }) as ISearchGeneric;
```

Comment:
@lukasolson, not really proud of this cast, but I did not know how to stop the type bleeding on me :(.

Reply:
I just played around again trying to get rid of this cast and it was fruitless, so I'm fine with this for now.